### ISSUE TITLE
Minifying CSS and Javascript templates

### ISSUE THREAD

**Issue by andrewbenton** [rejectedsoftware/diet-ng#26]
As a user and developer, I find that hand-written CSS and JS, both from files and filters, have a relatively heavy load compared to the already condensed HTML that diet produces.  I would like to be able to emit minified CSS and JS to the page.  Even something to strip indentation and spaces would contribute a lot.

**Comment by azbukagh** [comment]
You can use external tool for this. There is `preBuildCommands` in [dub.json](http://code.dlang.org/package-format?lang=json#build-settings), which are runned before building.

diet is just an another way to write HTML. It has nothing to do with CSS or JS.

**Comment by andrewbenton** [comment]
That's true, but being able to do the minification at runtime would also be useful and make the library more friendly to non-expert users of dub.
